.github/workflows/release.yml

```diff
@@ -55,3 +55,15 @@ jobs:
           repository: epinio/helm-charts
           event-type: epinio-release
           client-payload: '{"ref": "${{ steps.get_tag.outputs.TAG }}"}'
+
+      - name: Bump Homebrew formula
+        uses: mislav/bump-homebrew-formula-action@v1
+        if: "!contains(github.ref, '-')" # skip prereleases
+        with:
+          download-url: https://github.com/epinio/epinio/archive/refs/tags/${{ steps.get_tag.outputs.TAG }}.tar.gz
```

Comment:
The download url seems weird to me, because for example the x86 for darwin is :
https://github.com/epinio/epinio/releases/download/v0.6.1/epinio-darwin-x86_64

And I don't see tar.gz file published for version v0.6.0

Reply:
This is because it's the link of the sources ([https://github.com/epinio/epinio/archive/refs/tags/v0.6.1.tar.gz](https://github.com/epinio/epinio/archive/refs/tags/v0.6.1.tar.gz)).  
[Homebrew doesn't like binary formulae](https://docs.brew.sh/Acceptable-Formulae#we-dont-like-binary-formulae), they prefer to build the binaries from sources during installation, as you can see in the Formula: https://github.com/Homebrew/homebrew-core/blob/master/Formula/epinio.rb#L17-L21

I had a look at `goreleaser` as an example and they do the same, so they have the build approach for the core repo, while they use the binaries in their custom tap:

- https://github.com/Homebrew/homebrew-core/blob/master/Formula/goreleaser.rb
- https://github.com/goreleaser/homebrew-tap/blob/master/Formula/goreleaser.rb


Link to v0.6.0: https://github.com/epinio/epinio/archive/refs/tags/v0.6.0.tar.gz

Reply:
Thanks a lot for the clear explanation